### PR TITLE
[1605] - Banner extends: ability add to banner video instead of image.

### PR DESCRIPTION
**Pull request by Fant0m771**
Original Issue, this PR is going to fix: REPLACE WITH A LINK TO ISSUE ( publicly available )

Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature or this is a bug fix - use 8.x-2.x branch. We'll tag for release if the bug is critical asap or tag for release next bug fix release until critical issue arrived.

## Steps for review

Added some theming for Open Y Rose, please check in "Open Y Rose" theme for now.
- [ ] Run update database (updb or go to /update.php) on existing site.
- [ ] Go to banner edit.
- [ ] Check "Add video insted of image".
- [ ] Insert video.
- [ ] Save the node.

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: modules/openy_features/openy_prgf/modules/openy_prgf_banner/config/install/field.storage.paragraph.field_prgf_video.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+entity_type: paragraph
+type: entity_reference
+settings:
+  target_type: media
+module: core
+locked: false
+cardinality: 1
+translatable: true
+indexes: {  }
+persist_with_no_fields: false
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_banner/config/install/field.storage.paragraph.field_prgf_video.yml:18]: set everywhere to `true` please

File: themes/openy_themes/openy_rose/scss/modules/_paragraphs.scss [hunk 2]
```diff
@@ -490,6 +490,26 @@
         object-fit: cover;
       }
     }
+
+    @include breakpoint($screen-desktop) {
+      &.video {
```
Comment (podarok) [inline themes/openy_themes/openy_rose/scss/modules/_paragraphs.scss:495]: Is it possible to add this style to all OpenY themes?
Would be cool to keep sass and generated css in module and include it globally as it is already done for some other modules https://github.com/ymcatwincities/openy/blob/8.x-2.x/themes/openy_themes/scss/modules.scss
And included by every theme automatically for compiling

**Review by podarok — CHANGES_REQUESTED** [review]
Needs small love @Fant0m771 

@paige-kiecker this one is a feature/enhancement. Should we tag it into point release or wait for the 2.2?

**Comment by podarok** [comment]
Retry Deepcode

**Comment by podarok** [comment]
@Fant0m771 hey
Could you record a short screncast video for the feature? @paige-kiecker wants to know how it works before approving it for the OpenY release
Thank you in advance

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by podarok** [comment]
closing because of no input on this one
Open if there is a need

**Comment by podarok** [comment]
Closing as old 
Feel free to pick up in case if the feature is needed

**Pull request closed without merging**